pandas/tools/util.py

```diff
@@ -61,6 +61,13 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : str, numpy.dtype, or Python dtype, default None
```

Comment:
Yes, that's why I modified the comment.

I think `signed` and `unsigned` are acceptable.  I would leave `integer` out because there are already plenty of ways to specify (like `int`).